examples/perf.rs

```diff
@@ -7,20 +7,15 @@
 extern crate lewton;
 extern crate byteorder;
 
-fn main() {
-	match run() {
-		Ok(_) =>(),
-		Err(err) => println!("Error: {}", err),
-	}
-}
-
-use std::env;
 use lewton::VorbisError;
-use lewton::inside_ogg::OggStreamReader;
-use std::fs::File;
-use std::time::Instant;
 
+#[cfg(feature = "ogg")]
```

Comment:
Can you instead put `#![cfg(feature="ogg")]` to the top of the file?

Reply:
I've tried, but `cargo` can't find the main function in that way

Reply:
Oh I see. I think required-features is a better solution: https://doc.rust-lang.org/cargo/reference/manifest.html#the-required-features-field-optional

Reply:
Done, thanks!